com.unity.render-pipelines.universal/Runtime/LightCookieManager.cs

```diff
@@ -31,6 +31,8 @@ static class ShaderProperty
 
         private enum LightCookieShaderFormat
         {
+            None = -1,
+
```

Comment:
The extra line done on purpose or by accident?

Reply:
On purpose, since it's a bit special and enums often do not have negative values.